src/components/Search/SearchResultSeries.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Button, Container, Icon, Segment } from 'semantic-ui-react';
+
+import { selectors as lessonsSelectors } from '../../redux/modules/lessons';
+import { selectors as mdbSelectors } from '../../redux/modules/mdb';
+import { selectors as tagSelectors } from '../../redux/modules/tags';
+import { isDebMode } from '../../helpers/url';
+import Link from '../Language/MultiLanguageLink';
+import ScoreDebug from './ScoreDebug';
+import SearchResultBase from './SearchResultBase';
+import { CT_LESSONS_SERIES } from '../../helpers/consts';
+import SearchResultCollection from './SearchResultCollection';
+
+class SearchResultSeries extends SearchResultBase {
```

Comment:
Why not a hook? Everything new should be a hook.

Is it because of SearchResultBase class?

Reply:
yes, because of base class